atoMEC/staticKS.py

```diff
@@ -685,8 +712,18 @@ def calc_E_kin_unbound(orbs, xgrid):
                     config.mu[i], config.beta, 3.0
                 )
 
+        if config.unbound == "thomas_fermi":
+            E_kin_unbound_array = np.zeros((config.spindims, config.grid_params["ngrid"])) # initialize
+            E_kin_unbound = 0.0 # initialize
+            for i in range(config.spindims):
+                for j in range(len(v_s[0, :])):
+                    prefac = (2.0 / config.spindims) * sqrt(2) / (pi ** 2)
```

Comment:
Change to `prefac = (2.0 / config.spindims) / (sqrt(2) * pi ** 2)` to match `ideal` prefactor (mistake in notes sorry)